tests/spring-cloud-sleuth-instrumentation-reactor-tests/src/test/java/org/springframework/cloud/sleuth/instrument/web/client/ReactorNettyHttpClientBraveTests.java

```diff
@@ -82,13 +66,15 @@ static HttpClient testHttpClient() {
 	}
 
 	@Override
-	protected void closeClient(HttpClient client) {
-		// HttpClient is not Closeable
+	protected void closeClient(AnnotationConfigApplicationContext context) {
+		context.close(); // ensures shutdown hooks fire
```

Comment:
reactor hooks are a PreDestroy thing.. we formerly needed hacks because this test leaked spring contexts.